templates/server-serviceaccount-token.yaml

```diff
@@ -0,0 +1,21 @@
+{{/*
```

Comment:
Please can we rename the file to `server-serviceaccount-secret.yaml`? The `metadata.name` field makes sense to stick with -token, but I think we're fairly consistent about using the API object name in the file name.